web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
could you split this file in two file. (one file per model), and name the file ir_ui_view.py and ir_model.py ?